### PR TITLE
Adding NeMo Curator

### PR DESCRIPTION
**Pull request by exactlyallan**
Adding NeMo curator section per #376 request. Thanks for your patience. 
Some other minor content fixes.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,4 +1,8 @@
 github.com/FortAwesome/Font-Awesome v0.0.0-20230327165841-0698449d50f2/go.mod h1:IUgezN/MFpCDIlFezw3L8j83oeiIuYoj28Miwr/KUYo=
+github.com/FortAwesome/Font-Awesome v0.0.0-20240402185447-c0f460dca7f7/go.mod h1:IUgezN/MFpCDIlFezw3L8j83oeiIuYoj28Miwr/KUYo=
```
Comment (bdice) [inline go.sum:2]: Why does this have checksums for multiple versions? If we updated a dependency, we do not need to keep the old checksums for versions we no longer depend on. Is this automatically generated? Maybe delete it and regenerate it?

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "rapids.ai",
+  "name": "rapids.ai_ae",
```
Comment (bdice) [inline package-lock.json:2]: Why did this change?

**Review by bdice — CHANGES_REQUESTED** [review]
This PR looks like its git commit history is not quite right. Can you rebase these changes on the latest main branch, or otherwise fix the diffs that are showing up from previously-merged PRs?

File: package-lock.json (hunk 2 above)
Comment (exactlyallan) [reply package-lock.json:2]: I have multiple copies of RAPIDS repo from different forks.

**Comment by exactlyallan** [comment]
> This PR looks like its git commit history is not quite right. Can you rebase these changes on the latest main branch, or otherwise fix the diffs that are showing up from previously-merged PRs?

Yeah not sure what happened - I think Ill rebase and try again next change I have.

**Pull request closed without merging**